srcDLL/DllMain.cpp

```diff
@@ -60,11 +63,11 @@ BOOL WINAPI DllMain(HINSTANCE hInstance, DWORD dwReason, LPVOID reserved)
 		// Failure means op2ext.dll was loaded by something else, such as a unit test
 		// For unit tests, just stay in memory, it's not an error if this fails
 		if (EnableOp2MemoryPatching()) {
-			// Replace call to gTApp.Init with custom routine
-			// This hook is needed to further bootstrap the rest of module loading
+			// Initiall TApp event hooks
```

Comment:
Typo in comment. I think it should read "Initial"

Reply:
Oh geeze, I think I got confused between "initial" and "install". Thanks for pointing that out.